scripts/compose.py

```diff
@@ -803,10 +803,30 @@ def _content(self):
 
 class AgentGoNetHttp(Service):
     SERVICE_PORT = 8080
+    DEFAULT_AGENT_PACKAGE = "master"
```

Comment:
Python and Nodejs use `package`, Ruby uses `version`, and RUM uses  `branch`. IMHO should be only one way, I mean, you want to set the version/package/branch/whatever to use, if we have 3 different ways to set it you have to remember 3 different parameters. I will open a discussion about it. Here we can change it to `version` I like it more, I only choose package because was the most used

Reply:
https://github.com/elastic/apm-integration-testing/issues/226

Reply:
Sorry, didn't realise "package" was already being used as well.